packages/cli/lib/projects.js

```diff
@@ -608,7 +608,10 @@ const makePollTaskStatusFunc = ({
         ) + '\n';
 
     SpinniesManager.update(overallTaskSpinniesKey, {
-      text: `${statusStrings.INITIALIZE(taskName)}\n${componentCountText}`,
+      text: `${statusStrings.INITIALIZE(
+        taskName,
+        taskId
```

Comment:
Does` taskId` for deploys represent the deploy ID or the build ID?

Reply:
Ooh interesting, it's `deployId` here (Just checked on my end). Should I fix it to the correct build ID, or rewrite the text to display deploy ID?

Reply:
Fixed to correct build ID in https://github.com/HubSpot/hubspot-cli/pull/901/commits/0ccf74f6f549e3e5121bca2aff262e5f522f2d7d